messaging/relayer/internal/msgobserver/eth/observer/event_transformer.go

```diff
@@ -80,6 +81,11 @@ func (t *SFCEventTransformer) validate(event *functioncall.SfcCrossCall) error {
 	return nil
 }
 
+// getIDForEvent generates a deterministic ID for an event of the format {network_id}/{contract_address}/{block_number}/{tx_index}/{log_index}
+func (t *SFCEventTransformer) getIDForEvent(event types.Log) string {
```

Comment:
do we need to be scoped to the object `SFCEventTransformer`? as this can be a simple static function (utility in the file)

Reply:
yea, you are right. I initially had it as a static function instead of a method as well. I will change it back.